### PR TITLE
85 oauth partial

### LINKED ISSUE #85: Create a partial for google login button used on welcome page and learn more page

### PR DESCRIPTION
**Pull request by AngelaGuardia**
**Changes Made**

- Added a partial for the google OAuth login button 
- DRY'ed up the welcome view by refactoring the slides code

closes #85

### CODE REVIEW

File: app/views/learn_more/show.html.erb [hunk 1]
```diff
@@ -34,20 +34,7 @@
...
-        <h4 >Login with Google to:</h4>
-        <p class="w3-justify w3-center ">
-          - Set up a garden - <br>
-          - Track your sensor data - <br>
-          - Connect with your community - <br>
-          - Track your garden's carbon impact - <br>
-          - Track the health of your plants and soil -
-        </p>
-      </div>
+      <%= render 'shared/login_with_google_button' %>
```
Comment (dcoleman21) [inline app/views/learn_more/show.html.erb:37]: Did this change the styling at all?

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -18,72 +18,13 @@
...
-  <div class="mySlides w3-display-container w3-center">
-    <img src="https://i.imgur.com/ORgmfoY.jpg" style="width:916px">
-  </div>
-  <div class="mySlides w3-display-container w3-center">
-    <img src="https://i.imgur.com/IqmfCSG.jpg" style="width:916px">
-  </div>
-  <div class="mySlides w3-display-container w3-center">
-    <img src="https://i.imgur.com/uf6q7DV.jpg" style="width:916px">
-  </div>
+  <% images = ['ESOXGPn', 'JZzjPUH', 'y8P9c7q', '5gjnhlp', 'Yl4Zwic', 'Ewyct7X', 'E9nPGky', 'r1IZlFr', '43ja22n', 'v2jd9mo', 'rjBlkqp', 'yxPE2zw', 'vxkPqcp', 'RCxvJMX', '4EOtT70', 'MbOwjJK', 'LidCyvh', 'n4Rlajh', 'TiNcO3V', 'ORgmfoY', 'IqmfCSG','uf6q7DV']%>
```
Comment (dcoleman21) [inline app/views/welcome/index.html.erb:21]: Did this change the styling at all?

**Review by dcoleman21 — COMMENTED** [review]
I just wanted to make sure that they styling wasn't changed with any of this.

File: app/views/learn_more/show.html.erb (hunk 1 above)
Comment (AngelaGuardia) [reply app/views/learn_more/show.html.erb:37]: I checked it on localhost and it looked good. Admittedly I have not been exposed to the FE as much as you and Luke! So you may notice something I didn't.

Comment (LHJE) [reply app/views/learn_more/show.html.erb:37]: It all looks the same!  You're a wizard Angela :)

File: app/views/welcome/index.html.erb [hunk 3]
```diff
@@ -18,72 +18,13 @@
...
+
+  <% images.each do |image|  %>
+    <div class="mySlides w3-display-container w3-center">
+      <img src=<%= "https://i.imgur.com/#{image}.jpg" %> style="width:916px">
+    </div>
+  <% end %>
```
Comment (LHJE) [inline app/views/welcome/index.html.erb:27]: Oh my gosh this is SO clever.

**Review by LHJE — APPROVED** [review]
Angela you wizard. Nicely done

**Review by dcoleman21 — APPROVED** [review]
Brilliant.

**Pull request merged**